static/app/views/replays/detail/network/details/sections.tsx

```diff
@@ -101,7 +102,15 @@ export function QueryParamsSection({item}: SectionProps) {
 }
 
 export function RequestPayloadSection({item}: SectionProps) {
+  const {dismiss} = useDismissReqRespBodiesAlert();
+
   const hasRequest = 'request' in item.data;
+  useEffect(() => {
+    if (hasRequest) {
+      dismiss();
```

Comment:
Is this a no-op if it's already dismissed?

Reply:
turns out no. it'll bump the timestamp each time. 👎